src/components/Icons/icons.js

```diff
@@ -0,0 +1,16 @@
+const icons = {
+  "chevron-down": [
```

Comment:
I think we should move away from a single file containing all icons since this will lead to un-used icons making it into a final bundle.

Reply:
https://twitter.com/jxnblk/status/991345243520987136

Reply:
We convert all of our SVGs on the EDP to React components with svgo: https://github.com/svg/svgo